<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Concretos</class>
 <widget class="QDialog" name="Concretos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mm. de espesor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Proveedor</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mmEspesor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_agregar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Agregar Concreto</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_provedor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regresar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Regresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>% de desperdicio de mezcla</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_porcentajeDesperdicioMezcla">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>% de desperdicio de agua</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_metros3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Metros c√∫bicos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_porcentajeDesperdicioAgua">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
